autopush/diagnostic_cli.py

```diff
@@ -54,26 +75,35 @@ def run(self):
         md = match.groupdict()
         api_ver, token = md.get("api_ver", "v1"), md["token"]
 
-        parsed = self._conf.parse_endpoint(
-            self.db.metrics,
-            token=token,
-            version=api_ver,
-        )
-        uaid, chid = parsed["uaid"], parsed["chid"]
+        try:
+            parsed = self._conf.parse_endpoint(
+                self.db.metrics,
+                token=token,
+                version=api_ver,
+            )
+            uaid, chid = parsed["uaid"], parsed["chid"]
+        except (InvalidTokenException, InvalidToken) as ex:
+            print("Token could not be deciphered. Are you using the correct"
```

Comment:
is it worth including the exception in this message (I'm leaning towards sure, why not)?

Reply:
Yeah, was kinda on the fence. Mostly because the bulk was "Invalid Token", but yeah, there are times it's useful to dump as well.